extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -325,6 +342,32 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
             ></div>
           ))}
         </div>
+
+        {activeProductData.productDetails && (
+          <div class="i-amphtml-amp-story-shopping-pdp-details">
+            <button
+              class="i-amphtml-amp-story-shopping-pdp-details-header"
+              onClick={(e) => this.onDetailsHeaderClick_(e.target)}
+            >
+              <span class="i-amphtml-amp-story-shopping-pdp-details-header-text i-amphtml-amp-story-shopping-sub-section-header">
+                {this.localizationService_.getLocalizedString(
+                  LocalizedStringId_Enum.AMP_STORY_SHOPPING_ATTACHMENT_DETAILS,
+                  this.element
+                )}
+              </span>
+              <svg
+                viewBox="0 0 10 6"
+                class="i-amphtml-amp-story-shopping-pdp-details-header-caret"
+              >
+                <path />
```

Comment:
Is this `</path>` needed?

Reply:
Yep, the element is needed so we can [set the path and animate](https://github.com/processprocess/amphtml/blob/details/extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css#L282-L289) it with CSS.